Extensions/BBText/JsExtension.js

```diff
@@ -0,0 +1,508 @@
+        _('Set text alignment'),
+        _('Set text alignment'),
+        _('Set text alignment of _PARAM0_ to _PARAM1_'),
+        '',
+        'res/actions/textAlign24.png',
+        'res/actions/textAlign24.png'
+      )
+      .addParameter('object', _('BBText object'), 'BBText', false)
+      .addParameter(
+        'stringWithSelector',
+        _('Alignment'),
+        '["left", "right", "center"]',
+        false
+      )
+      .getCodeExtraInformation()
+      .setFunctionName('setAlignment')
+      .setGetter('getAlignment');
+
+    object
+      .addAction(
+        'SetWidth',
+        _('Set wrapping width'),
+        _('Set wrapping width'),
+        _('Set wrapping width of _PARAM0_ to _PARAM1_ _PARAM2_'),
+        '',
+        'res/actions/scaleWidth24.png',
+        'res/actions/scaleWidth24.png'
+      )
+      .addParameter('object', _('BBText object'), 'BBText', false)
+      .addParameter('operator', _("Modification's sign"), '', false)
+      .addParameter('expression', _('Number'), '', false)
+      .getCodeExtraInformation()
+      .setFunctionName('setWrappingWidth')
+      .setManipulatedType('number')
+      .setGetter('getWrappingWidth');
+
+    // extension
+    //   .addExpression(
+    //     'GetText',
+    //     _('Get text'),
+    //     _('Get text'),
+    //     _('BBText'),
+    //     '',
+    //     'JsPlatform/Extensions/bbcode32.png',
+    //     'JsPlatform/Extensions/bbcode32.png'
+    //   )
+    //   .getCodeExtraInformation()
+    //   .setFunctionName('getBBText');
+
+    // object
+    //   .addCondition(
+    //     'Pause',
+    //     _('Is paused'),
+    //     _('Check if the video is paused.'),
+    //     _('_PARAM0_ is paused'),
+    //     '',
+    //     'JsPlatform/Extensions/videoicon24.png',
+    //     'JsPlatform/Extensions/videoicon16.png'
+    //   )
+    //   .addParameter('object', _('Example object'), 'ObjectExample', false)
+    //   .getCodeExtraInformation()
+    //   .setFunctionName('isPaused');
+
+    return extension;
+  },
+  /**
+   * You can optionally add sanity tests that will check the basic working
+   * of your extension behaviors/objects by instanciating behaviors/objects
+   * and setting the property to a given value.
+   *
+   * If you don't have any tests, you can simply return an empty array like this:
+   * `runExtensionSanityTests: function(gd, extension) { return []; }`
+   *
+   * But it is recommended to create tests for the behaviors/objects properties you created
+   * to avoid mistakes.
+   */
+  runExtensionSanityTests: function(gd, extension) {
+    return [];
+  },
+  /**
+   * Register editors for objects.
+   *
+   * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change.
+   */
+
+  registerEditorConfigurations: function(objectsEditorService) {
+    objectsEditorService.registerEditorConfiguration(
+      'BBText::BBText',
+      objectsEditorService.getDefaultObjectJsImplementationPropertiesEditor()
```

Comment:
For add help button when you edit the text in IDE
`objectsEditorService.getDefaultObjectJsImplementationPropertiesEditor({
        helpPagePath: "/objects/BBTextObject"
      })`

Reply:
@Bouh thank you 😄 I will add those after work. Do you have any idea why my getters are not appearing in expressions in the event sheet? Am I using the setGetter method wrong?

Reply:
I don't know maybe `getBBText `not working, uncomment the expression for test `getBBText`, expression are without or` setGetter()`

Reply:
Just to be sure there is no misunderstanding: `setGetter` is not creating automatically expressions. You still have to call `addExpression` (or `addStrExpression`). 
setGetter is only here to define the JavaScript method that will be on the left hand side of the operator when using + (or -, / or * for numbers): `object.setter(object.getter + value)`
for example: `object.setBBText(object.getBBText() + value)`.

Reply:
@4ian sorry I did misunderstood it would somehow magically and quickly add the getters for me. I will do it manually like before :) No shortcuts